intro/matplotlib/examples/plot_multiplot_ex.py

```diff
@@ -5,24 +5,25 @@
 Show multiple subplots in matplotlib.
 """
 
-import pylab as pl
+import numpy as np
```

Comment:
And here too?

I am asking this because in the logic of being didactic, I think that not importing something not used is better (less cognitive overall).